apps/settings/js/panels/operator_settings/models/operator_manager.js

```diff
@@ -142,21 +142,22 @@ define(function(require) {
 
     this.debug('search');
     this._searching = true;
-    return this._connWrapper.search().then((networks = []) => {
-      var operators = networks.map((network) => OperatorItem(network));
-      operators.forEach((operator) => {
+    return this._connWrapper.search().then(function(networks = []) {
```

Comment:
& please update jsdoc version to 3.3.2 in `gaia/package.json` first to check if the issue has been solved.